tests/dbutils.py

```diff
@@ -6,16 +6,16 @@
 # TODO: should this be somehow be divined from environment?
 POSTGRES_USER, POSTGRES_HOST = 'postgres', 'localhost'
 
+TEST_DB_NAME = '_test_db'
 
 def db_connection(dbname=None):
     conn = psycopg2.connect(user=POSTGRES_USER, host=POSTGRES_HOST,
         database=dbname)
     conn.autocommit = True
     return conn
 
-
 try:
-    conn = db_connection()
+    conn = db_connection(dbname=None)
```

Comment:
Calling with explicit `None`, not `TEST_DB_NAME`, because test db not yet created